paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/samples/networking/Models.kt

```diff
@@ -56,12 +60,91 @@ data class ExampleCheckoutResponse(
     } else {
         null
     }
+}
+
+@Serializable
```

Comment:
So much gson! Where is the kotlinx.serialization!?!?

I've got a few spare cycles I could spend updating our sample if you think it's an alright change.

Reply:
Feel free to update this afterwards.